platform/android/export/export.cpp

```diff
@@ -261,7 +261,7 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		EditorProgress *ep = nullptr;
 	};
 
-	Vector<PluginConfig> plugins;
+	Vector<AndroidPluginConfig> plugins;
```

Comment:
I think we'd typically use `PluginConfigAndroid` to further specify `PluginConfig` (see `EditorExportPlatformAndroid`).

Which raises the question of whether this shouldn't be made abstract as a `PlatformPluginConfig` interface that each platform can implement, when relevant? Though that's outside the scope of this PR.